### DISCUSSION TITLE
v2.15.0

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#3230] [General]
## What's Changed
* Feature: Add color choices to NavigationItem badges by @JackWH in https://github.com/filamentphp/filament/pull/3218
* Fix: Floating UI Table Panels Issues by @awcodes in https://github.com/filamentphp/filament/pull/3184
* feature: Navigation group objects by @danharrin in https://github.com/filamentphp/filament/pull/3216
* feature: Reorderable tables by @danharrin in https://github.com/filamentphp/filament/pull/3211
* Fixed translated direction problem by @alperenersoy in https://github.com/filamentphp/filament/pull/3228
* ✨ `filament/notifications` package by @zepfietje in https://github.com/filamentphp/filament/pull/3091

## New Contributors
* @JackWH made their first contribution in https://github.com/filamentphp/filament/pull/3218

**Full Changelog**: https://github.com/filamentphp/filament/compare/v2.14.5...v2.15.0

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v2.15.0'>v2.15.0</a>.</em>

**Comment by sajjadhossainshohag** [comment]
Hello Dan Darrin,
( Cannot use object of type Filament\Navigation\NavigationGroup as array) this type of error after the upgrade.

**Reply by danharrin** [reply]
Ill reply on #3233
